config/initializers/edition_services.rb

```diff
@@ -7,7 +7,10 @@
 Whitehall.edition_services.subscribe("unpublish") { |event, edition, options| ServiceListeners::SearchIndexer.new(edition).remove! }
 Whitehall.edition_services.subscribe(/^(force_publish|publish)$/) { |_, edition, _| Whitehall::PublishingApi.publish(edition) }
 Whitehall.edition_services.subscribe(/^(force_publish|publish)$/) { |_, edition, _| EditionDependenciesPopulator.new(edition).populate! }
+Whitehall.edition_services.subscribe(/^(force_publish|publish)$/) { |_, edition, _| edition.republish_dependent_editions }
+Whitehall.edition_services.subscribe(/^(force_publish|publish)$/) { |_, edition, _| edition.destroy_dependency_records }
```

Comment:
@heathd: in response to the conversation over Slack, i've made destruction of dependency records explicit, such that it can be visible from `edition_services.rb`. thanks for pointing that out!